salt/__init__.py

```diff
@@ -115,38 +115,29 @@ def __define_global_system_encoding_variable__():
     # and reset to None
     encoding = None
 
-    if not sys.platform.startswith("win") and sys.stdin is not None:
-        # On linux we can rely on sys.stdin for the encoding since it
-        # most commonly matches the filesystem encoding. This however
-        # does not apply to windows
+    if sys.stdin is not None:
+        # We can rely on sys.stdin for the encoding since it most commonly
+        # matches the filesystem encoding
         encoding = sys.stdin.encoding
 
     if not encoding:
```

Comment:
Is there really ever a case where encoding would be Falsey here?

Reply:
I can't think of one... no... I was just keeping the original code.